authorization/model/organization_model_service_blackbox_test.go

```diff
@@ -0,0 +1,104 @@
+package model_test
+
+import (
+	"testing"
+
+	"github.com/fabric8-services/fabric8-auth/account"
+	"github.com/fabric8-services/fabric8-auth/authorization"
+	"github.com/fabric8-services/fabric8-auth/authorization/model"
+	"github.com/fabric8-services/fabric8-auth/authorization/resource"
+	"github.com/fabric8-services/fabric8-auth/gormtestsupport"
+
+	"github.com/fabric8-services/fabric8-auth/authorization/common"
+	"github.com/fabric8-services/fabric8-auth/authorization/role"
+	"github.com/satori/go.uuid"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+)
+
+type organizationServiceBlackBoxTest struct {
+	gormtestsupport.DBTestSuite
+	repo             resource.ResourceRepository
+	identityRepo     account.IdentityRepository
+	identityRoleRepo role.IdentityRoleRepository
+	resourceRepo     resource.ResourceRepository
+	resourceTypeRepo resource.ResourceTypeRepository
+	roleRepo         role.RoleRepository
+	orgService       authorization.OrganizationService
```

Comment:
The name of the file is `organization_model_service_blackbox_test.go` but yours tests test the `OrganizationService` contract.

`organization_model_service_blackbox_test.go` should have the `OrganizationModelService` contracts tested.
`organization_service_blackbox_test.go` should have the `OrganizationService` contracts tested.

Reply:
I've renamed and reorganized these tests, there is some duplication between the service and the model service tests, however I suppose that it doesn't hurt anything.